.github/workflows/release_gh.yaml

```diff
@@ -179,38 +179,48 @@ jobs:
         with:
           ref: ${{ inputs.sha }}
       # Download all artifacts to current working directory
-      - uses: actions/download-artifact@v4
```

Comment:
are these downloading all artifacts? if so, wouldn't it be easier to just put it on a path and do
```      
    - name: Download all artifacts
        uses: actions/download-artifact@v4
        with:
          path: whatever
```